enterprise/reporting/reporting.go

```diff
@@ -259,6 +259,7 @@ func (r *DefaultReporter) getReports(currentMs int64, syncerKey string) (reports
 
 func (*DefaultReporter) getAggregatedReports(reports []*types.ReportByStatus) []*types.Metric {
```

Comment:
@itsmihir I think we can rename `getReports` -> `getAggregatedReports` and probably remove the old `getAggregatedReports` function ?

Reply:
In `getAggregatedReports`, we continue to aggregate reports, and merging both functions would yield a significantly larger function.